Assets/Scripts/Networking/PlayerManager.cs

```diff
@@ -272,6 +272,11 @@ public void AnimateShoot()
           
         }
 
+        private void PlayShootingSound()
+        {
+            GameObject.Find("PaintGun").GetComponent<AudioSource>().Play();
```

Comment:
We should put the sound source on the camera. Adding it on the gun might make the gun sound louder or quieter depending on how high up the gun is held, which doesn't make much sense.

Reply:
It's so close, that there's actually no difference I think

Reply:
I changed it though, so it's a bit less risky finding just AudioSource in the gameObject itself and not PaintGun when the sound difference is so small